content/learn/book/welcome/plugins/_index.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Modularity through plugins"
+weight = 3
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+In Bevy, virtually all of the functionality you might want is added via **plugins**, which are added to your app.
+These might provide basic windowing functionality, handle input or sound, calculate physics or provide game-specific logic like a tile map system.
+
+By combining Bevy's first-party plugins with third-party alternatives, you can modularly customize the behaviour of the Bevy game engine, and then add your own plugins to contain your own game-specific code in a well-organized way.
+
+## Writing your own plugins
+
+Plugins are collections of code that modify the `App` (which controls all of the data and logic of our game) using the builder pattern.
```

Comment:
Also as a general rule: if you are referencing a "rust symbol", I think it should use the "rust short code link", even if you've already provided a link elsewhere. I want the Bevy Book to feel "strongly typed".